tests/core/full_node/stores/test_block_store.py

```diff
@@ -264,3 +266,40 @@ def generator(i: int) -> SerializedProgram:
             assert await store.get_generator(blocks[4].header_hash) == new_blocks[4].transactions_generator
             assert await store.get_generator(blocks[6].header_hash) == new_blocks[6].transactions_generator
             assert await store.get_generator(blocks[7].header_hash) == new_blocks[7].transactions_generator
+
+
+class TestBlockStoreAPI:
```

Comment:
I don't think you should put these tests inside a `class`. I think it's been a mistake that we have done this with existing tests